pom.xml

```diff
@@ -209,15 +211,19 @@
         <libpam4j.version>1.11</libpam4j.version>
         <log4j.version>1.2.17</log4j.version>
         <log4j2.version>2.12.1</log4j2.version>
+        <lombok-version>1.18.4</lombok-version>
+        <log4j2.version>2.12.1</log4j2.version>
```

Comment:
Looks like merge/copy paste error? Duplicate `log4j2.version` property

Reply:
Why are we pulling in lombok here? This isn't used across the other features within Knox and I don't see any reason to add it here making this code different from the rest.

We do need to see a contributed consumer of this within the PR that makes it usable out of the box for the Knox community.

Reply:
Sure, will remove Lombok to keep the code consistent.